internal/cmd/cmd_test.go

```diff
@@ -54,39 +54,39 @@ func TestCompile(t *testing.T) {
 	assertDoCompileFiles(
 		t,
 		false,
-		`testdata/compile/dep_errors.proto:6:1:Expected ";".`,
+		`testdata/compile/dep_errors.proto:6:1:PROTO Expected ";".`,
```

Comment:
You do not want to print this out. This is a `protoc` error, and the goal is to just convert them to `filename:line:column:message`. The logic that prints this out should be removed.

Reply:
I see why this is printed out actually - we'll discuss. I think you're right that everything should have an ID, but see discussion below of what an ID is, we'll discuss offline.